packages/node_modules/@webex/plugin-meetings/src/meeting-info/index.js

```diff
@@ -48,14 +49,15 @@ export default class MeetingInfo extends StatelessSparkPlugin {
   /**
    * Fetches meeting info from the server
    * @param {String} destination one of many different types of destinations to look up info for
+   * @param {String} type optional type to match up with the destination value
    * @returns {Object} returns a meeting info object
    */
-  fetchMeetingInfo(destination) {
-    const options = {
-      destination,
-      type: SIP_URI
-    };
-
-    return this.meetingInfoRequest.fetchMeetingInfo(options).then((info) => this.setMeetingInfo(info.id, info));
+  fetchMeetingInfo(destination, type = null) {
+    MeetingInfoUtil.conversationServiceUrl = this.spark.internal.device.services.conversationServiceUrl;
```

Comment:
Why are we setting the conversation service url for `MeetingInfoUtil` in this file?

Reply:
to build the conversation url -- im open to ideas here this was my *first* off the cuff idea, i wasn't a huge fan of doing it in this way, we can discuss in standup